### PR TITLE
feat: update LoadingPlaceholder to support Heart theme

### PR DESCRIPTION
**Pull request by jasononeil**
Note about Zen fallback:
- The "Zen" fallbacks are actually updated in this PR to match the Zen UI Kit.
- Zen default bg was Wisteria 100, now Wisteria 200
- Zen reversed bg was white 50% opacity, now white 65% opacity

The pattern I've used to add alpha to the token _only if using the custom property_
relies on cascading fallbacks in CSS Custom Properties.

I did a codepen to demonstrate how this works:
https://codepen.io/jasononeil/pen/vYyjzMz?editors=1100

Alternatively, I could use React Context to get the theme, but that won't work with the Elm version.

:shrug:

# Objective

# Motivation and Context

# Screenshots (if appropriate)

# Checklist

- [ ] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

File: draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@import "~@kaizen/design-tokens/sass/color";
+
+// Option tokens for Heart theme background colours.
+$ot-color-background-slate-10: rgba(var(--kz-var-color-slate-rgb-params), 0.1);
```
Comment (ActuallyACat) [inline draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss:4]: I like this naming, I'm finding it pretty easy to reason about.

File: draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+$ot-color-background-white-10: rgba(var(--kz-var-color-white-rgb-params), 0.1);
+
+// Decision tokens for background colours based on presence of CSS variables.
+// If the `--kz-var-color-*` variable our `--ot-*` variables depend on is missing, the Zen tokens are used as a fallback.
+$dt-color-background-default: var(
```
Comment (ActuallyACat) [inline draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss:9]: Super surprised that fallbacks can work within an `rgba` function like this. Very cool.

**Review by ActuallyACat — APPROVED** [review]
LGTM

File: draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss (hunk 1 above)
Comment (sebpearce) [reply draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss:4]: I am a bit confused about seeing both "option token" and "background" to refer to the same thing. My understanding was that option tokens are devoid of meaning (like background colour) and decision tokens are what assigns an option token to a meaning. Have I got a weird take?

File: draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss (hunk 2 above)
Comment (jasononeil) [reply draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss:21]: Me too! I needed a codepen to convince myself it was real https://codepen.io/jasononeil/pen/vYyjzMz?editors=1100

File: draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss (hunk 1 above)
Comment (jasononeil) [reply draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss]: I hadn't seen these comments and went and refactored to try and make it clearer.

@ActuallyACat can you take another look?

Comment (jasononeil) [reply draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss]: @sebpearce your understanding is correct, what I'm doing is just a bit weird, and the naming was off - this was just supposed to represent Slate (as defined by a CSS custom property) with 10% alpha. A true option token shouldn't have had the word "background". Perhaps `ot-color-slate-opacity-10`?

Either way I've had a go at refactoring so the raw CSS variable names are not used outside of the decision tokens file (rather, a mixin exists that you add to the component root which defines the css variables that the decision tokens depend on)

**Review by ActuallyACat — DISMISSED** [review]

File: draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/styles.scss [hunk 3]
```diff
@@ -1,14 +1,15 @@
-@import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/color-vars";
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/component-library/styles/animation";
```
Comment (ActuallyACat) [inline draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/styles.scss:3]: Can we remove this import and swap the variables to `design-tokens/sass/animation-vars`? 
@kaizen/component-library/styles/animation

**Review by ActuallyACat — APPROVED** [review]

**Review by ActuallyACat — APPROVED** [review]

**Pull request merged**